pkg/kubelet/apis/cri/services.go

```diff
@@ -43,6 +43,8 @@ type ContainerManager interface {
 	ListContainers(filter *runtimeapi.ContainerFilter) ([]*runtimeapi.Container, error)
 	// ContainerStatus returns the status of the container.
 	ContainerStatus(containerID string) (*runtimeapi.ContainerStatus, error)
+	// UpdateContainerResources updates the cgroup resources for the container.
+	UpdateContainerResources(containerID string, resources *runtimeapi.LinuxContainerResources) error
```

Comment:
Another option is to mirror the protobuf message and define a new struct like ContainerUpdate which for now has only a LinuxContainerResources member. Or just reuse the generated protobuf type.

Note: this merged already.